pmemkv/src/main/java/site/ycsb/db/pmemkv/PmemKVClient.java

```diff
@@ -177,7 +181,7 @@ public Status read(final String table, final String key, final Set<String> field
   @Override
   public Status scan(final String table, final String startkey, final int recordcount, final Set<String> fields,
                      final Vector<HashMap<String, ByteIterator>> result) {
-    // TODO(kfilipek): Implement if possible/necessary
+    /* TODO(kfilipek): Implement if possible/necessary */
```

Comment:
maybe remove the name? Just TODO?

Reply:
1. [TODO Comments](https://google.github.io/styleguide/cppguide.html#TODO_Comments)
I use TODO comments with name according to GCS for C++. Using reference for someone, issue ID or something is okey for me. Always you can create issue for this part of implementation (mock) and reference here issue ID instead of the user name.
2. [GCS for Java](https://google.github.io/styleguide/javaguide.html#s4.8.6-comments)
Google Coding Style for Java didn't specify how to comment and what is preferred, but IMO changing comments from one-line to the block comments (for one-line comments) are unnecessary effort.

Reply:
ad 1. for me it's whatever ;)

@KFilipek, as for the link you've pasted - there's a sentence in this guide, which you will hate:
`Each line of text in your code should be at most 80 characters long.`
soo.. I guess we don't follow this guide fully :D

ad 2. maybe it's unnecessary, but it's already done ;) also, for me, it's a matter of consistency, thanks to that change, we use one style of comments in the file ;)